paasta_tools/cli/cmds/cook_image.py

```diff
@@ -77,6 +77,7 @@ def paasta_cook_image(args, service=None, soa_dir=None):
             service=service,
             loglevel='debug'
         )
+        print output
```

Comment:
Do you still want this? a debug log-level should already go to stdout.

Although, we should not really _log if we are doing a local-run if you ask me.